editor/src/core/shared/template-path.ts

```diff
@@ -60,9 +60,17 @@ export function clearTemplatePathCache() {
   globalScenePathCache = emptyScenePathCache()
 }
 
-function getScenePathCache(sceneElementPath: StaticElementPath): ScenePathCache {
+function getScenePathCache(sceneElementPaths: StaticElementPath[]): ScenePathCache {
+  let joinedPaths: string[] = []
+  for (var i = 0, len = sceneElementPaths.length; i < len; i++) {
```

Comment:
Why is `len` initialised as part of the `for` loop? I thought for a moment it was being changed in here.

Reply:
I've inlined our `fastForEach` here because I needed access to the index

Reply:
As for why I didn't use `fastForEach` since that also provides the index, I don't know...